src/main/java/io/github/mountainrange/mule/WorldMap.java

```diff
@@ -0,0 +1,14 @@
+package io.github.mountainrange.mule;
+
+/**
+ * Holds information about the entire map.
+ */
+public class WorldMap {
+
+	Tile[][] tiles;
```

Comment:
It's a skeleton right now; we should put all the information about terrain and who owns what in here (see also `Tile`).